### PR TITLE
Fix/intro grammar

### PR DESCRIPTION
**Pull request by kevinmpowell**
I've used https://hemingwayapp.com to edit the intro section for conciseness. 

Here's the [before](https://deploy-preview-74--dtcg-tr.netlify.app/format/#status) and [after](https://deploy-preview-76--dtcg-tr.netlify.app/format/?utm_source=github&utm_campaign=bot_dp#status) for comparison.

References #66

### CODE REVIEW

File: technical-reports/format/index.html [hunk 1]
```diff
@@ -38,27 +38,29 @@
...
+      </p>
+      <p>
+        The
         <a href="https://www.w3.org/community/about/process/cla">
           Community Contributor License Agreement (CLA) </a
-        >, as specified by the
+        >governs contributions to this draft, as specified by the
         <a href="https://www.w3.org/community/about/process/#cgroups"
           >W3C Community Group Process</a
         >.
```
Comment (kaelig) [inline technical-reports/format/index.html:66]: This section is boilerplate from the W3C that you'll generally find untouched or very lightly edited in drafts. Not sure we should optimize for readability or consistency.

**Comment by kaelig** [comment]
I agree with the intent, but think an over-reliance on Hemingway is detrimental to the readability and clarity of this doc.

I find that following Hemingway's suggestions too closely leads to a succession of tiny sentences that can read as quite dry, and breaks the flow of otherwise very readable content. In one of the sentences, all jargon was removed in an undesirable lossy rewrite.

My guidance is to use Hemingway sparingly, and take its suggestions for what they are: suggestions.

👍🏻 It's useful to avoid very long sentences, repetitions, and spot passive voice that would read better as active.

👎🏻 It warns about jargon that's actually important in our context: technical reports are aimed at experts who rely on this vocabulary to derive precise meaning.

---

(Grammarly allows to specify the target audience, tone, level of formality… I personally find it better than Hemingway for professional writing like technical proposals, and still use Hemingway for blog posts)

File: technical-reports/format/index.html (hunk 1 above)
Comment (c1rrus) [reply technical-reports/format/index.html:66]: I'd lean towards consistency in this instance. After all, we're trying to align with W3C's tooling, process and writing style elsewhere.

File: technical-reports/format/index.html [hunk 2]
```diff
@@ -74,57 +76,20 @@ <h1>Introduction</h1>
...
-        <li>
-          Documentation tools can display design token names rather than the raw
-          values in design specs and style guides.
-        </li>
-      </ul>
-      <p>
-        It is often desirable for design system teams to integrate such tools
-        together, so that design token data can flow between design and
-        development tools. For example:
+        Design tokens establish a common vocabulary across organizations. They
```
Comment (c1rrus) [inline technical-reports/format/index.html:79]: Whoa! Harsh edit :-P

I'm all for making things more concise and using plain English, but I think the specific examples of tools that might use tokens are useful here to set context.

File: technical-reports/format/index.html [hunk 3]
```diff
@@ -74,57 +76,20 @@ <h1>Introduction</h1>
...
+        are a method for expressing design decisions.
       </p>
-      <ul>
```
Comment (c1rrus) [inline technical-reports/format/index.html:106]: If we leave the above bit as is, then we need to keep this list too.

File: technical-reports/format/index.html [hunk 4]
```diff
@@ -74,57 +76,20 @@ <h1>Introduction</h1>
...
-        own, bespoke "glue" code or workflows. Furthermore, if teams want to
-        migrate to different tools, they will need to update those integrations.
+        Many tools allow importing and exporting design tokens, but formats are
+        tool-specific. The burden of sharing tokens across tools is on design
+        system teams. They must create and maintain their own, bespoke "glue"
+        code or workflows. Migrating to different tools means updating those
+        integrations.
       </p>
       <p>
-        This specification aims to facilitate better interoperability between
```
Comment (c1rrus) [inline technical-reports/format/index.html:124]: I think the original is better here. The edited version is too vague and "largest goal" just sounds weird to me.

**Review by c1rrus — CHANGES_REQUESTED** [review]
I'd revert most of these edits TBH. They're stripping away too much info IMHO.

**Comment by kevinmpowell** [comment]
Good feedback. Declining PR.

**Pull request closed without merging**